lmfdb/website.py

```diff
@@ -145,6 +145,11 @@ def robots_txt():
         fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), "static", "robots.txt")
         if os.path.exists(fn):
             return open(fn).read()
+    # not running on www.lmfdb.org
+    else:
+        fn = os.path.join(os.path.dirname(os.path.abspath(__file__)), "static", "default_robots.txt")
+        if os.path.exists(fn):
+            return open(fn).read()
     return "User-agent: *\nDisallow: / \n"
```

Comment:
well, the final return statement is no longer hit. 

the idea here was that any other lmfdb setup is signalling to not be indexed. this should lead to establishing the main lmfdb.org website as the sole one to be indexed and there is no duplicated page at a random place elsewhere on the web. search engines detect this, and then they think this is just a non-unique content being ripped off and somehow decides who is the real one and who is not. that might lead to being removed from the index for the main site. 

tl;dr: in case anyone has concerns about SEO, the idea is to disallow indexing any copy of the main lmfdb page for all search engines

Reply:
> well, the final return statement is no longer hit.

I agree with you, I left it there, just in case either file doesn't exist.